modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/paths/EVPUtils.path

```diff
@@ -15,6 +15,16 @@
 	<td>//div//a[contains(@class,'btn-solid') and contains(.,'Sign In')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>KEBAB_MENU_OPTION_ORGANIZATION</td>
+	<td>//span[contains(@class,'organizationName')][contains(text(),'${key_name}')]//following-sibling::div[@id="evp-organization-kebab-button"]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>KEBAB_OPTION</td>
+	<td>//div[contains(@class,'menu-open')]//span[contains(text(),'${key_option}')]</td>
+	<td></td>
+</tr>
```

Comment:
Hi @juliosilveira77, this path isn't necessary. You can use `MenuItem#ANY_MENU_ITEM`, that already exists.